web-bundle/src/main/java/com/graphhopper/http/HttpErrorsOnlyFilterFactory.java

```diff
@@ -0,0 +1,20 @@
+package com.graphhopper.http;
+
+import ch.qos.logback.access.spi.IAccessEvent;
+import ch.qos.logback.core.filter.Filter;
+import ch.qos.logback.core.spi.FilterReply;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import io.dropwizard.logging.filter.FilterFactory;
+
+@JsonTypeName("http-errors-only")
+public class HttpErrorsOnlyFilterFactory implements FilterFactory<IAccessEvent> {
```

Comment:
Is this class used or was it just for demo-purposes?

Reply:
No this one can be used to filter the request logs (showing only errors).